cmd/export/main.go

```diff
@@ -17,37 +17,102 @@ package main
 
 import (
 	"context"
-	"log"
+	"fmt"
 	"net/http"
+	"time"
 
 	"go.opencensus.io/plugin/ochttp"
 
 	"github.com/google/exposure-notifications-server/internal/export"
 	"github.com/google/exposure-notifications-server/internal/logging"
 	_ "github.com/google/exposure-notifications-server/internal/observability"
+	"github.com/google/exposure-notifications-server/internal/serverenv"
 	"github.com/google/exposure-notifications-server/internal/setup"
 )
 
 func main() {
 	ctx := context.Background()
 	logger := logging.FromContext(ctx)
 
+	if err := realMain(); err != nil {
+		logger.Fatal(err)
+	}
+}
+
+func realMain() error {
+	ctx := context.Background()
+
+	config, env, closer, err := doSetup()
+	if err != nil {
+		return fmt.Errorf("setup: %w", err)
+	}
+	defer closer()
+
+	server, err := NewServer(ctx, config, env)
+	if err != nil {
+		return fmt.Errorf("newserver: %w", err)
+	}
+
+	go server.Run()
+	return nil
+	// return server.Stop()
+}
+
+func doSetup() (*export.Config, *serverenv.ServerEnv, setup.Defer, error) {
```

Comment:
I don't like this, but I can't think of a better way to individually test "config/environment parsing".